AN00209_xCORE-200_DSP_Library/app_fft_double_buf/src/app_fft_double_buf.xc

```diff
@@ -16,13 +17,14 @@ mechanism accessing a shared memory area.
 
 The task produce_samples fills one buffer with a stereo signal
 whilst the do_fft task processes the other buffer
-"in place" which means the buffer is also used to store the real part of the result
-Tasks access these buffers via *movable* pointers. These pointers can
-be safely transferred between tasks without any race conditions
-between the tasks using them.
+"in place" which means the buffer is also used to store the real part of the
+result Tasks access these buffers via *movable* pointers. These pointers can be
```

Comment:
There was a missing full stop after result, which was less important with the original line break arrangement.